### DISCUSSION TITLE
[Grant] Metaplex DAO | Grants Working Group

### DISCUSSION THREAD

**Discussion by MetaplexFoundation** [metaplex-foundation/dao#9] [Community Grant Submissions]
**Submission details**

Submitter: SoCalStreet
Email: [deanslistgrants@gmail.com)
Submitter wallet: Hsd1M2KrYFQNkJtErH2WBwscgUuPYYM7zitZXPmPGr5L
Beneficiary wallet: HRLdZUitTR6ekSWqR8VqmYnEDVnDy8jhkJPB24t9hoB7
$MPLX requested: 1,200,000

**Team**

The team will be a 3-4 person team and consist of core members of the Dean's List community who have been administering the current governance related grants program - Decentragrants.

We would also propose the inclusion of a member of our Dean's List community who currently works with the Metaplex Foundation as an outside advisor. This role, while only advisory, would have significant value in terms of ensuring our applicants have a direct "line in" to the core Metaplex Foundation team and any trouble shooting / guidance / support required.

The Metaplex DAO Working Group may consider additional advisors at its discretion to serve as subject matter experts on each of the two focus areas described herein.

**Abstract**

This proposal is for creating a Metaplex DAO Grants Working Group and launching two targeted grants initiatives.

The working group will establish an effective, efficient, and scalable process for Metaplex DAO grants, focusing on promoting and adopting Metaplex CORE and the MPL-404 hybrid DeFi standard. Dean’s List, an established Solana power user community, will manage these programs.

We request 1,200,000 MPLX tokens from Metaplex DAO. If the DAO approves our grant submission, the Metaplex Foundation has agreed to contribute $150,000 in USDC to support the grant program.

**Description**

The core responsibilities of the working group will be as follows:

Process + Evaluation:

- Processing, vetting, and indexing/cataloging all inbound grant applications.

- This includes initial screening and follow up DD calls with the Grants Council team for all applicants fitting the mandate of the grant programs.

- Implementing a standardized application and information disclosure process to simplify community evaluation and to create a "starting point" for establishing best practices around standards for applicant disclosure for future grants initiatives.

Administration:

- The creation and maintenance of a grant dashboard to track all grants across the full life cycle of the grants.

- This will include post completion tracking of key adoption/usage metrics to verify efficacy of capital allocation.

Educational Content Creation:

- For applicants that engage in initial DD, the working group will produce standardized, educational/informational material about each applicant to enhance understanding within the DAO, support more informed decision-making, and to create the foundation of a content library around the Metaplex grants program.

Community Engagement and Awareness:

- The working group will seek to build an active early MPLX Grants community with the aim to drive awareness and participation in the grant initiatives.

- In practice, this will mean organizing and hosting community events with applicants to highlight program launches (e.g., Twitter Spaces, DAO-only Discord town-halls)

- We will also conduct monthly Grants ecosystem calls to provide updates, promote applicants, and enhance overall awareness.

- Outside of highlighting applicants, we will also look to facilitate events (Twitter Spaces/Discord town halls) with investors and builders from the Metaplex ecosystem, to discuss potential use cases and ideas for things to build.

Governance and Best Practices:

- The grants working group will ensure that the Metaplex DAO benefits from ongoing experiments in governance best practices, often led by team members from the Dean’s List team and/or Realms.

**Milestones (optional)**

Budget + Timeline

This proposal is for a 6 month targeted grant program and consists of two pools of capital USDC and MPLX tokens. Any MPLX not delivered to grantees would be returned to the DAO at the end of the 6 months.

Metaplex CORE Grant Pool Request
US$50,000 in USDC
500,000 locked MPLX tokens to be distributed with 1 year lock.

Metaplex MPL 404 Grant Pool Request
US$50,000 in USDC
500,000 locked MPLX tokens to be distributed with 1 year lock.

Administrative Fee
US$50,000 in USDC
200,000 in MPLX tokens, to be distributed with 1 year lock

This proposal requests 1,200,000 MPLX tokens from the Metaplex DAO.

The Metaplex Foundation has agreed to contribute the requested US$150,000 in USDC to support the grant program in the event the DAO approves this proposal.

**Comment by dannpl** [comment]
MPL-404 is the future of NFT!

we are looking forward to migrating our SPL-404 system to the MPL-404

@triadxyz

**Comment by MetaplexFoundation** [comment]
This grant proposal was passed by the Metaplex DAO.
